### PR TITLE
Vertical scrolling support

### PR DESCRIPTION
**Pull request by tejuamirthi**
This is a fix is for the vertical scrolling support: https://github.com/superpeteblaze/ScalingCarousel/issues/78
Please review and let me know @superpeteblaze

### CODE REVIEW

File: ScalingCarousel/Classes/ScalingCarouselView.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@ open class ScalingCarouselView: UICollectionView {
     // MARK: - Properties (Private)
 
     private var lastCurrentCenterCellIndex: IndexPath?
-
+    public var scrollDirection: UICollectionView.ScrollDirection = .horizontal
```
Comment (superpeteblaze) [inline ScalingCarousel/Classes/ScalingCarouselView.swift:23]: To match the access level of other 'public' properties, can we change this to `open`, allowing it to be overridden if someone wants to do that

Also, minor, but can we move this down into the `MARK: - Properties (Public)` section below.

File: ScalingCarousel/Classes/ScalingCarouselView.swift [hunk 2]
```diff
@@ -205,9 +219,13 @@ fileprivate extension PrivateAPI {
...
          ScalingCarouselView
         */
-        invisibleScrollView.heightAnchor.constraint(equalTo: heightAnchor).isActive = true
-        invisibleScrollView.topAnchor.constraint(equalTo: topAnchor).isActive = true
-        
+        if scrollDirection == .horizontal {
+            invisibleScrollView.heightAnchor.constraint(equalTo: heightAnchor).isActive = true
+            invisibleScrollView.topAnchor.constraint(equalTo: topAnchor).isActive = true
+        } else {
+            invisibleScrollView.widthAnchor.constraint(equalTo: widthAnchor).isActive = true
```
Comment (superpeteblaze) [inline ScalingCarousel/Classes/ScalingCarouselView.swift:226]: To hide the invisibleScrollView's scrolling indicator, please add the following at line 196 above:

```
invisibleScrollView.showsVerticalScrollIndicator = false
```

**Review by superpeteblaze — CHANGES_REQUESTED** [review]
This is great, thanks for the contribution @tejuamirthi.  I've tired it out and it works well. 

Just two suggested changes for the PR.

File: ScalingCarousel/Classes/ScalingCarouselView.swift (hunk 1 above)
Comment (tejuamirthi) [reply ScalingCarousel/Classes/ScalingCarouselView.swift]: Hi @superpeteblaze,
I made it open and moved it under the MARK - PUBLIC properties.  
Thank you for pointing out minor things as well

**Comment by tejuamirthi** [comment]
I have updated with requested changes. 
Let me knoe if anything else

**Review by superpeteblaze — APPROVED** [review]
Looks good to me 👍🏽

**Pull request merged**